test/models/shelter_test.rb

```diff
@@ -6,4 +6,19 @@ class ShelterTest < ActiveSupport::TestCase
     assert_equal Shelter.count, 2
   end
 
+  test "shelter name is NOT nil" do
+    shelters = Shelter.all
```

Comment:
All we're really testing here is that the fixtures set a shelter name. Not sure it provides much value. I think remove this test for now.